src/server/session.ts

```diff
@@ -1599,6 +1599,10 @@ namespace ts.server {
                 return emptyArray;
             }
 
+            function dtsChangeCanAffectEmit(compilationSettings: CompilerOptions) {

```

Comment:
Since this depeneds on only compilerOptions which is parameter, move this outside the class so we are not creating local function

Reply:
Moved.